src/Pages/auth/Login.tsx

```diff
@@ -0,0 +1,11 @@
+import AuthSection from "../../components/AuthSection/AuthSection";
```

Comment:
It seems logical to me that if we want to create a reusable component, the AuthSection should not be responsible for determining the content of the sign up and login pages, as this would make the Login and Sign Up pages unnecessary. Instead, it would make more sense to include the common elements in the AuthSection component and merge the contents of the LoginForm and SignUpForm into the Login and Sign Up pages, respectively.